### DISCUSSION TITLE
How to rotate the GND map by 90°?

### DISCUSSION THREAD

**Discussion by BettyWalzier** [mtex-toolbox/mtex#718] [Ask Anything] [answered]
Hi all,

Recently, I am working on the GND density of FCC metal. The plotting code is as follows:

**plot(ebsdS,GND,'micronbar','off')
mtexColorMap('jet')
mtexColorbar
set(gca,'ColorScale','log'); % this works only starting with Matlab 2018a
set(gca,'CLim',[1E13 1E16]);
hold on
plot(grains.boundary,'linewidth',1)
hold off**

The result is good, but I want to rotate the GND map 90° counterclockwise. 
Is there anyone who knows how to do it?

Thanks in advance!
Best regards,
Betty

**Comment by ralfHielscher** [comment, marked as answer]
you may try `plotx2east` or `plotx2north` at the beginning of your script.

Ralf.

**Reply by BettyWalzier** [reply]
Hi Ralf.,

Thanks for your kind reply. I will try in this way.
I have another question want to ask: 
**Can the EBSD data be divided evenly using MTEX?**  Like the function in Channel 5 as below:
![image](https://user-images.githubusercontent.com/76571114/105514263-92761880-5cd3-11eb-8d67-13b1e24c4ccb.png)

Thanks in advance!
Best regards,
Luqing Cui

**Comment by kilir** [comment]
Hi,
`ebsd.extent` gives you the min/max of x,y coordinates, hence you can easily derive the size and coordinates of any tiling. Then use `EBSD/inpolygon`.

https://mtex-toolbox.github.io/EBSD.extend.html
https://mtex-toolbox.github.io/EBSD.inpolygon.html

Cheers,
Rüdiger
